File: service-backbase-java/src/main/java/org/example/backend/model/response/UserResBody.java
package org.example.backend.model.response;

import pl.net.testit.serum.commons.json.JsonEntity;

public class UserResBody  extends JsonEntity {

  public User user;

  public static class User {

    public String email;
    public String token;
    public String username;
    public String bio;
    public String image;
  }

}

File: service-backbase-kotlin/src/main/kotlin/org/example/backend/model/response/UserResBody.kt
package org.example.backend.model.response

class UserResBody(
  val user: UserResBodyUser? = null,
)

class UserResBodyUser(
  val email: String? = null,
  val token: String? = null,
  val username: String? = null,
  val bio: String? = null,
  val image: String? = null,
)